### PR TITLE
UG and DG for sales component

### PR DESCRIPTION
**Pull request by HyperionNKJ**
DeveloperGuide.adoc:

Added user stories for sales.
Added use cases for sales.
Added non-functional requirement: System should not require any internet access except for updating

UserGuide.adoc

Added commands for sales.

### CODE REVIEW

File: docs/DeveloperGuide.adoc [hunk 1]
```diff
@@ -875,6 +887,182 @@ Steps 1b1-1b2 are repeated until the number of units entered is less than the nu
...
+
+1. User requests to record sales volume of a menu item within a specified day.
+2. App appends the record at the end of the day's sales report.
+3. App returns a success message confirming that the record is successful.
++
+Use case ends.
+
+*Extensions*
+[none]
+* 1a. Invalid command format entered by the user
```
Comment (AZhiKai) [inline docs/DeveloperGuide.adoc]: Example of missing full stop.

File: docs/DeveloperGuide.adoc [hunk 2]
```diff
@@ -891,6 +1079,7 @@ _{More to be added}_
 .  Should always favour security over efficiency in development.
 .  Any user who has the basic proficiency of the English language should be able to use the application with the help
  of the link:{repoURL}/docs/UserGuide.adoc[`UserGuide.adoc`].
+.  System should not require any internet access except for updating
```
Comment (AZhiKai) [inline docs/DeveloperGuide.adoc]: Example of missing full stop.

File: docs/UserGuide.adoc [hunk 3]
```diff
@@ -480,6 +480,116 @@ Format: `history [stockup] [consumption]`
...
+=== Display sales report: `display-sales`
+
+Displays the sales report for all days +
+Format: `display-sales` +
+OR +
+Displays the sales report for the specified day +
+Format: `display-sales DATE`
+
+****
+* DATE must be written in the DD/MM/YYYY format
```
Comment (AZhiKai) [inline docs/UserGuide.adoc]: Not necessary. Take a look at this library: https://github.com/nus-cs2103-AY1819S1/forum/issues/32

**Comment by AZhiKai** [comment]
Also, please fill in the details at this issue so we all know what we're doing for each milestone. :)

**Comment by HyperionNKJ** [comment]
Done adding the full stops! :)
I was actually considering using Natty when coming up with the features. But I felt it will complicate things up (Like unnecessary time and future dates)

**Comment by AZhiKai** [comment]
> Done adding the full stops! :)
> I was actually considering using Natty when coming up with the features. But I felt it will complicate things up (Like unnecessary time and future dates)

Ah I see. Okay, do whatever you deem necessary for your feature! 👍

**Comment by AZhiKai** [comment]
Nice work, @HyperionNKJ!
Your PR has been approved and merged.

Everyone, please pull from upstream to update your local `master` branch, and then merge `master` into your own branch if you're still doing stuffs.

**Pull request merged**